### ISSUE TITLE
Bug: Emoji picker menu items are inaccessible via keyboard

### ISSUE THREAD

**Issue by nickytonline** [open-sauced/app#2411]
### Describe the bug

Emoji picker menu items are inaccessible via keyboard

### Steps to reproduce

1. Ensure you're logged in to the https://app.opensauced.pizza.
2. Go to the highlights feed, https://app.opensauced.pizza/feed
3. Click on the smiley icon then press tab
4. Press <kbd>Shift</kbd> + <kbd>TAB</kbd>, and the smiley icon gets focus (a good thing)
5. Press <kbd>ENTER</kbd>. The emoji picker menu opens.
6. Press <kbd>TAB</kbd>.
7. Notice that the first emoji in the list does not receive focus. The focus goes to the next focusable element, a link.

![CleanShot 2023-12-29 at 13 11 36](https://github.com/open-sauced/app/assets/833231/810bee6d-7abe-42f8-99ab-4b057e3592ff)

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### Contributing Docs

- [X] I agree to follow this project's Contribution Docs

**Comment by nickytonline** [comment]
The emojis in the emoji picker should be buttons but still retain the current styling aside from a focus state, which they currently do not have as they are `<divs />`.

The focus state will get enabled as soon as you make the emoji items buttons thanks to the global focus state styles.

**Comment by nickytonline** [comment]
Actually, for the focus state for these buttons, you'll need to add the following to globals.css.

```diff
...

+ button:not([role]) {
+   @apply border;
+   @apply border-transparent;
+ }
+ 
  input[type="checkbox"]:focus-visible,
  [role="checkbox"]:focus-visible,
  [role="radio"]:focus-visible,
  [role="combobox"]:focus-visible,
- [role="switch"]:focus-visible {
+ [role="switch"]:focus-visible,
+ button:not([role]):focus-visible {
    @apply outline-none;
    @apply ring;
    @apply border-orange-500;
    @apply ring-orange-100;
  }

...
```

**Comment by nickytonline** [comment]
Also, once #2409 is merged, let's move the `onClick` on the containing `<div />` for the emoji items to an `onClick` on the buttons.

**Comment by CBID2** [comment]
.take

**Comment by CBID2** [comment]
Hey @nickytonline! :) I made the PR. It's in draft mode for now, so you can review and add more suggestions.

**Comment by nickytonline** [comment]
I'm on vacation, but if it's ready for review, take it out of draft.

I'll take a peek at it when I'm back on the 8th unless someone else does before me.

You can use my video in the issue as the before in your PR and just provide an after video with it working via keyboard.
